src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -11,12 +12,26 @@ internal class EndToEndTraceHelper
         private static string appName;
         private static string slotName;
 
-        // TODO: Replace TraceWriter with structured event client
-        private readonly TraceWriter appTraceWriter;
+        private readonly ILogger logger;
 
-        public EndToEndTraceHelper(TraceWriter appTraceWriter)
+        public EndToEndTraceHelper(JobHostConfiguration config)
         {
-            this.appTraceWriter = appTraceWriter;
+            string instrumentationKey = Environment.GetEnvironmentVariable("APPINSIGHTS_INSTRUMENTATIONKEY");
```

Comment:
Why is Application Insights needed here? Why can't we just use the Azure Functions ILogger pipeline? Do we really have to have this? I can imagine this is hard to manage a long while.

Reply:
I'll look into this.  Thanks!